lib/booker/v4/models/special.rb

```diff
@@ -0,0 +1,54 @@
+module Booker
+  module V4
+    module Models
+      class Special < Model
+        attr_accessor 'ID',
```

Comment:
I think attributes that are objects need to have nested classes, e.g., 'SpecialItems', 'Brands', 'FreeItems'